aytests/double_users.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e -x
+zgrep "Found users in profile with equal <username>."\
```

Comment:
You don't need to use `&&` if you want that line shorter (putting `echo "AUTOYAST OK"` in a separate line is fine because of `set -e`). But it's fine as it is right now.